genesyscloud/provider/sdk_client_pool.go

```diff
@@ -505,7 +507,7 @@ func wrapWithRecover(method resContextFunc, operation constants.CRUDOperation) r
 // Inject a pooled SDK client connection into a resource method's meta argument
 // and automatically return it to the Pool on completion
 func runWithPooledClient(method resContextFunc) resContextFunc {
-	return func(ctx context.Context, r *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	return func(ctx context.Context, r *schema.ResourceData, meta any) diag.Diagnostics {
```

Comment:
Why are you setting `meta` to an `any`?

Reply:
Just to improve code readability. There no functional difference between the two, but `any` looks a lot better imo